app/javascript/controllers/item_selector_controller.js

```diff
@@ -130,4 +154,19 @@ export default class extends Controller {
       `;
     }).join('');
   }
+
+  getStimulusParams(element) {
+    const params = {}
+    const pattern = new RegExp(`^data-${this.identifier}-(.+)-param$`, "i")
+
+    for (const { name, value } of Array.from(element.attributes)) {
+      const match = name.match(pattern)
```

Comment:
would it not work to iterate over `element.dataset`?

Reply:
This is copied directly from stimulus 🤷‍♂️

Reply:
oh wild. well, can't argue with that, I guess